src/Internal.AspNetCore.Sdk.Tasks/Internal.AspNetCore.Sdk.Tasks.csproj

```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <DefineConstants>$(DefineConstants);SDK</DefineConstants>
+    <Description>Package expressly designed to carry tasks used by Internal.AspNetCore.Sdk. This package should never be directly referenced.
+    Use Internal.AspNetCore.BuildTools.Tasks if build tasks are required.</Description>
```

Comment:
Shouldn't everything except KoreBuild depend on the `Internal.AspNetCore.Sdk` package? `Internal.AspNetCore.BuildTools.Tasks` is a special case that's not relevant here.